server/src/main/kotlin/org/javacs/kt/index/SymbolIndex.kt

```diff
@@ -48,47 +82,49 @@ private object FqNames : Table() {
 class SymbolIndex {
     private val db = Database.connect("jdbc:h2:mem:symbolindex;DB_CLOSE_DELAY=-1", "org.h2.Driver")
 
-    var progressFactory: Progress.Factory = Progress.Factory.None
+    var progressFactory: Progress.Factory = object: Progress.Factory {
+        override fun create(label: String): CompletableFuture<Progress> = CompletableFuture.supplyAsync { Progress.None }
+    }
```

Comment:
Any reason for not using `Progress.Factory.None`? If it's important that the future completes asynchronously, we may want to update `Progress.Factory.None` itself since I don't think we use it elsewhere.

Reply:
Yeah, `Progress.Factory.None` makes more sense 👍